### ISSUE TITLE
Update Risk Case Event Order Validation: Incorporate AML_SAR in Description and Logic

### ISSUE THREAD

**Issue by rpallapolu** [ground-truth-ai/amlaidatatests#72]
Description says, Validates order of risk case events: AML_PROCESS_START, AML_EXIT, AML_PROCESS_END occur in exactly this order for each risk_case_id, party_id, However SQL query actually checks for AML_SAR order as well. Can you please update the description  as well as logic that align with the feedback below?

We can have the AML_SAR  occur after AML_EXIT, but both AML_SAR and AML_EXIT events are in between AML_PROCESS_START and AML_PROCESS_END.
Example: AML_START < (AML_SAR, AML_EXIT in any order) < AML_END

https://github.com/ground-truth-ai/amlaidatatests/blob/c82dc91ac010b869d72f0c4c9d280c48329c467b/generated_sql/bigquery/DT014-EventOrder-type_risk_case_event.sql#L17

**Comment by ssabdb** [comment]
Agreed, this was on our original specification, but I can't see any errors in the [data validation errors](https://cloud.google.com/financial-services/anti-money-laundering/docs/reference/data-validation-errors).
